bundles/org.openhab.binding.sbus/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0
+                        https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Switch Device -->
+	<thing-type id="switch">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="udp"/>
+		</supported-bridge-type-refs>
+		<label>SBUS Switch</label>
+		<description>SBUS switch device</description>
+		<config-description>
+			<parameter name="subnetId" type="integer">
```

Comment:
These configuration parameters are not listed in the README.md. Maybe create a markdown table and record the options. Look at other bindings how that can be very clear to the end-user.
Also applies to the other Things declared in this file.